src/EditorFeatures/Core.Wpf/InheritanceMargin/MarginGlyph/MultipleMembersMarginViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.VisualStudio.Imaging.Interop;
+
+namespace Microsoft.CodeAnalysis.Editor.InheritanceMargin.MarginGlyph
+{
+    internal class MultipleMembersMarginViewModel
```

Comment:
It's probably worth having a base for the single and multiple view model. There's some common things you'll need, and can help clean up the XAML binding a bit too

Reply:
With some change recommendations from https://github.com/dotnet/roslyn/pull/52145/files/bcc29c22993f09095d353a12ad62f503b58a6f2b#r603649567 we should be able to collapse the view models. The UI is Menu -> Menu Items, and menu items can have child menu items. We should be able to reflect that in the view models as well

Reply:
I am guessing this is based on your previous comment. : ) If there is a good way to bind to the property I am happy to extract them.

Reply:
Done. : )